src/Services/InventoryUpdateService.php

```diff
@@ -32,14 +33,16 @@ class InventoryUpdateService
    */
   private function validateInventoryRequestData($inventoryRequestDTO, $loggerContract): bool
   {
-
     if (!isset($inventoryRequestDTO)) {
       return false;
     }
 
+    $issues = [];
+
     if ($inventoryRequestDTO->getQuantityOnHand() < 0) {
```

Comment:
I believe we allow suppliers to indicate -1 quantity for cases such as Made To Order (MTO) products and explicitly discontinuing products. I can reach out to confirm but I'm not sure if you had more context around this validation step?

Reply:
This was hidden in the sync code and I moved it into this Validation helper function for "separation of concerns" reasons. You're probably the 3rd or 4th person (myself included) to think that it is weird code.
The number being evaluated comes from Plentymarkets, so this is most likely protection against passing a negative number from Plentymarkets (for whatever reasons they would use negative stock) and accidentally indicating something to Wayfair.

Reply:
I think we might want to allow -1 to pass through from PM to Wayfair. I can confirm with the Inventory team that this is a valid value they want to receive.

Reply:
to reiterate: my concern would be that Plentymarkets reason for providing a negative number would not (always) match up with Wayfair's understanding of what it meant.
I understand we'd have legitimate reasons to send -1, but we would probably need to do that problematically instead of letting a negative number pass-thru from PM to Wayfair.

Reply:
yeah it looks like version 1.1.1 DID allow the -1 to pass through!
Once we've got confirmation, I'll change it back.